gh-138307: Update the Ellipsis documentation (#138306)
python/cpython: Doc/library/constants.rst

.. index:: single: ...; ellipsis literal
.. data:: Ellipsis

   The same as the ellipsis literal "``...``", an object frequently used as a

>    placeholder for another value. Assignment to  ``Ellipsis`` is possible, but

Reviewer: I'm not sure "a placeholder for another value" is a good summary. I don't think it describes many of the typing use cases, for one.

Maybe "to indicate that something is omitted"?

>    assignment to  ``...`` is not possible and raises a :exc:`SyntaxError`.

Reviewer 2: ```suggestion
   assignment to  ``...`` raises a :exc:`SyntaxError`.
```

>    placeholder of another values. Assignments to  ``Ellipsis`` are possible.
>    However, assignments to  ``...`` are illegal and raise a :exc:`SyntaxError`.

Reviewer 3: ```suggestion
   placeholder for another value. Assignment to  ``Ellipsis`` is possible, but
   assignment to  ``...`` is not possible and raises a :exc:`SyntaxError`.
```
